src/tests/topp/NucleicAcidSearchEngine_11_out.idXML

```diff
@@ -16,7 +16,7 @@
 				<UserParam type="string" name="label" value="UGAGGUAGUAGGUUGUAUAGU"/>
 				<UserParam type="float" name="OMS:precursor_mz_error_ppm" value="-1.851164946914477"/>
 				<UserParam type="int" name="isotope_offset" value="2"/>
-				<UserParam type="float" name="q-value" value="0.0"/>
+				<UserParam type="float" name="PSM-level q-value" value="0.0"/>
```

Comment:
so why is this stored as user param and not as CV term (`"MS:1002354", "PSM-level q-value",`) ?

Reply:
Because PeptideHit is derived from MetaInfoInterface, but has no facility for storing CV terms.